obspy/core/util/attribdict.py

```diff
@@ -131,11 +131,6 @@ def __getattr__(self, name, default=None):
     def copy(self):
         return copy.deepcopy(self)
 
-    def __deepcopy__(self, *args, **kwargs):  # @UnusedVariable
```

Comment:
Is this `self.__dict__.update(self.defaults)` in `__setstate__` necessary? It's already in `__init__`. Are these `__get/setstate__` methods necessary?

Reply:
No idea, to be honest.. but I guess if tests keep passing we might be OK to remove them? dunno